FindPOLine/utils.js

```diff
@@ -59,7 +59,7 @@ export const buildOrderLinesQuery = (queryParams, isbnId, normalizedISBN, locale
       [FILTERS.RECEIPT_DUE]: buildDateRangeQuery.bind(null, `physical.${FILTERS.RECEIPT_DUE}`),
       [FILTERS.CLAIM_SENT]: buildDateRangeQuery.bind(null, [FILTERS.CLAIM_SENT]),
       [FILTERS.TAGS]: buildArrayFieldQuery.bind(null, [FILTERS.TAGS]),
-      [FILTERS.FUND_CODE]: buildArrayFieldQuery.bind(null, FILTERS.FUND_DISTRIBUTION),
+      [FILTERS.FUND_CODE]: (filterValue) => `fundDistribution =/@fundId (${filterValue.join(' or ')})`,
```

Comment:
Have you checked it for the case when 1 value is selected?

Reply:
Do you mean there may be a case where the selection is not multi-select?

Reply:
I mean that query broke the request with one selected fund, because the filter will return a string instead of an array